packages/react-popover/.storybook/preview.js

```diff
@@ -4,4 +4,4 @@ import * as rootPreview from '../../../.storybook/preview';
 export const decorators = [...rootPreview.decorators];
 
 /** @type {typeof rootPreview.parameters} */
-export const parameters = { ...rootPreview.parameters, layout: 'centered' };
```

Comment:
Is it expected?

Reply:
ah yeah good catch. it only really affects local canvas. The main docs already ignores this setting

Reply:
we might need to do AST parsing in the future for this kind of stuff :/